cmd/version/version.go

```diff
@@ -17,7 +17,10 @@ var (
 )
 
 // https://docs.ci.openshift.org/docs/getting-started/useful-links/#services
-const releaseURL = "https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream/4-stable/latest"
+const (
+	releaseURL          = "https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream/4-stable/latest"
+	multiArchReleaseURL = "https://multi.ocp.releases.ci.openshift.org/api/v1/releasestream/4-stable-multi/latest"
```

Comment:
ideally the multi payload is the only one we would need to use

Reply:
Yeah. I think Jeff and I have talked about that in passing as well. I removed the `multiArchReleaseURL` and just updated `releaseURL` with the multi-arch URL.